### ISSUE TITLE
Release latest version to fix `expand_arg`

### ISSUE THREAD

**Issue by wmmc88** [eupn/macrotest#94]
I was trying to setup `macrotest` and found that `expand_arg` was not working to pass my feature flags to the tests. This seems to be an issue with version "1.0.9",  because pointing macrotest to the latest commit on main [c4151a5](https://github.com/eupn/macrotest/commit/c4151a5f9f545942f4971980b5d264ebcd0b1d11) fixed to issue.

The output i got when on "1.0.9" was:
```
Running 1 macro expansion tests
Expansion error:
error: Package `package v0.0.0 (path/to/package)` does not have the feature `nightly`
```

**Comment by wmmc88** [comment]
not sure if related, but I also get the same errors when using workspace metadata in the `cargo.toml`: ex. `edition.workspace = true`

**Comment by ijackson** [comment]
Another bug fixed on master: #98

**Comment by wmmc88** [comment]
@eupn @taiki-e can we get a crates.io release of macrotest?

**Comment by taiki-e** [comment]
> @eupn @taiki-e can we get a crates.io release of macrotest?

(I'm not a maintainer of this crate, so I cannot release a new version. That said I would be happy to help with the maintenance of this crate if @eupn needs help on maintenance.)

**Comment by wmmc88** [comment]
> > @eupn @taiki-e can we get a crates.io release of macrotest?
> 
> (I'm not a maintainer of this crate, so I cannot release a new version. That said I would be happy to help with the maintenance of this crate if @eupn needs help on maintenance.)
> 

Woops. My bad. I saw the recent prs and misread it. Thought u were the one who merged them.

**Comment by eupn** [comment]
@taiki-e thanks for the offer, I've sent you an invite on github and crates.io! :3

**Comment by taiki-e** [comment]
@eupn Thanks! I'll work on a new release.

**Comment by taiki-e** [comment]
Published v1.0.10.
